srcalc/src/main/java/gov/va/med/srcalc/domain/model/DiscreteTerm.java

```diff
@@ -135,6 +135,6 @@ public int hashCode()
     public String toString()
     {
         return String.format(
-                "%s=%s?->%s", getVariable(), getOption(), getCoefficient());
+                "%s=%s->%s", getVariable(), getOption(), getCoefficient());
```

Comment:
The question mark was intentional here: it was supposed to imply a boolean condition.